modules/specialized-guides/pages/public-cloud-guide/payg/payg-overview.adoc

```diff
@@ -1,13 +1,17 @@
 = {payg} overview
-:description: {productname} {payg} or {payglongform} is a flexible and cost-effective solution that allows users to manage and monitor their systems. Unlike traditional subscription models, {payg} does not require a long-term subscription. Users can leverage the images of {productname} on a {payglongform} basis, paying only for the time of use, and the number of managed and monitored systems. {payg} with {productname} is an optimal solution for those seeking control, convenience, and cost savings when managing and monitoring their infrastructure on AWS, Azure, and GCP.
```

Comment:
It would be good to backport the PAYG changes to 4.3.  "traditional" here does not mean the traditional contact method ;)